pkg/docker/docker.go

```diff
@@ -134,8 +135,13 @@ func (l *Launchable) Launch(_ *runit.ServiceBuilder, _ runit.SV) error {
 	default:
 		return util.Errorf("invalid restart policy: %s", l.RestartPolicy_)
 	}
+
+	binds := []string{
+		fmt.Sprintf("/data/pods/%s:/data/pods/%s", l.RunAs, l.RunAs),
```

Comment:
I notice we hard code '/data/pods` in a few places. I wonder if it makes sense to declare constants or variables with that represent these common paths?

Reply:
Also should these be filepaths instead of strings?

Reply:
And they should be strings if you see https://github.com/esuen/p2/blob/add-docker-support/vendor/github.com/docker/docker/api/types/container/host_config.go#L333.